certified-connectors/Data8/apiDefinition.swagger.json

```diff
@@ -1956,6 +2010,148 @@
           }
         ]
       }
+    },
+    "/EmailValidation/CleanseSimple.json": {
+      "post": {
+        "consumes": [
+          "application/json"
+        ],
+        "responses": {
+          "200": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "Status": {
+                  "type": "object",
+                  "properties": {
+                    "Success": {
+                      "type": "boolean",
+                      "description": "Flag to determine if the request to Data8 succeeded, regardless of result",
+                      "title": "Success",
+                      "enum": [
+                        "",
```

Comment:
The type of the enum value is 'String' but it should be 'Boolean', Kindly correct this

Reply:
Resolved